### PR TITLE
Adding options to the VIP Go Node Preflight Checks command

### PR DESCRIPTION
**Pull request by saroshaga**
## Description

This PR adds the following command options for the VIP Go Node Preflight Checks command:
- `--wait=3000` or `-w 3000`: Time in milliseconds to wait for command to execute (default is 3000)
- `--verbose`: Increase logging level to include app build and server boot up messages
- `--help` or `-h`: command usage guide (default is false)

## Steps to Test

1. Check out PR.
1. Run `npm run build`.
1. Go to your target node app and run the following in the app's root directory `node <path_to_preflight_checks_repo>/dist/index.js` 
1. Verify that all the command options work as expected

### CODE REVIEW

**Review by chrean — APPROVED** [review]
To me this looks great and a great addition to the package.
Waiting for a more valuable review by @elazzabi .

**Review by elazzabi — APPROVED** [review]
Wooot wooot tested locally and it looks 🔥 

Thank you!

**Pull request merged**
